<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordsOnlineDialog</class>
 <widget class="QDialog" name="WordsOnlineDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <wstring>Dialog</wstring>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblLang">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <wstring>Language:</wstring>
       </property>
       <property name="buddy">
        <cstring>cboLang</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboLang">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="modelColumn">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblDict">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <wstring>Dictionary:</wstring>
       </property>
       <property name="buddy">
        <cstring>cboDict</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboDict">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="modelColumn">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QWebEngineView" name="wvOnline" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="url" stdset="0">
      <url>
       <wstring>about:blank</wstring>
      </url>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblWord">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <wstring>Word:</wstring>
       </property>
       <property name="buddy">
        <cstring>edtWord</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtWord">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <wstring>一人</wstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <wstring>Go</wstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtWord</tabstop>
  <tabstop>btnGo</tabstop>
  <tabstop>wvOnline</tabstop>
  <tabstop>cboLang</tabstop>
  <tabstop>cboDict</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>edtWord</sender>
   <signal>returnPressed()</signal>
   <receiver>btnGo</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
